health/health.d/entropy.conf

```diff
@@ -15,5 +15,8 @@ component: Cryptography
     every: 5m
      warn: $this < (($status >= $WARNING) ? (200) : (100))
     delay: down 1h multiplier 1.5 max 2h
-     info: minimum number of entries in the random numbers pool in the last 5 minutes
+     info: Minimum number of entries in the random numbers pool in the last 5 minutes. \
+           A low number of entries in the random numbers pool. \
```

Comment:
This is entirely wrong.

The alarm is checking the minimum number of _bits_ of entropy available.